src/main/java/model/School.java

```diff
@@ -0,0 +1,63 @@
+package model;
+
+import lombok.AccessLevel;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+@Getter
+@Setter
+@EqualsAndHashCode
+public class School {
+    @Setter(AccessLevel.NONE)
+    private final String id;
+    private String name;
+    private HashMap<String, Teacher> teachers = new HashMap<>();
+    private HashMap<String, Course> courses = new HashMap<>();
+    private HashMap<String, Student> students = new HashMap<>();
+
+    public School(String name) {
+        id = UUID.randomUUID().toString();
+        this.name = name;
+    }
+
+    public void addCourse(Course course) {
+        courses.put(course.getCourseId(), course);
+    }
+
+    public void addTeacher(Teacher teacher) {
+        teachers.put(teacher.getTeacherId(), teacher);
+    }
+
+    public void addStudent(Student student) {
+        students.put(student.getStudentId(), student);
+    }
+
+    public double getTotalProfit() {
+        return getTotalEarned() - computeSalaries();
+    }
+
+    private double computeSalaries() {
+        double totalSpent = 0;
+        for (Map.Entry<String, Teacher> entry : teachers.entrySet()) {
+            Teacher teacher = entry.getValue();
+            double spent = teacher.getSalary();
+            totalSpent += spent;
+        }
+        return totalSpent;
+    }
+
+    private double getTotalEarned() {
```

Comment:
Here as in the previous method, we could simplify using streams. It could be something like that..
    private double getTotalEarned() {
        return courses.values().stream()
                .mapToDouble(c -> c.getMoney_earned())
                .sum();
    }
Or even better as..
    private double getTotalEarned() {
        return courses.values().stream()
                .mapToDouble(Course::getMoney_earned)
                .sum();
    }